Presentation.Web/Content/less/kitos.less

```diff
@@ -1677,3 +1677,18 @@ tbody.bordered > tr > td {
         font-weight: 400;
     }
 }
+
+.kendo-margin-right {
+    margin-right: 0.16em;
+}
+
+.kendo-margin-left {
+    margin-left: 0.16em;
+}
+
+.kendo-btn-sm {
+    padding: 3.5px 10px;
+    font-size: 14px;
+    line-height: 1.5;
```

Comment:
did you figure out what the default int for this was?
If not, let's find out if it is `pt` er `em` and then be explicit about it :-)